src/main/java/lotto/domain/RanksCount.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import lotto.enums.WinningRank;
+
+import java.util.*;
+
+public class RanksCount {
+    private static final int INIT = 0;
+    private static final int MATCHED_COUNT_MIN = 3;
+    private static final int PLUS = 1;
+    private final TreeMap<WinningRank, Integer> ranksCount;
```

Comment:
```suggestion
    private final Map<WinningRank, Integer> ranksCount;
```
여기 타입은 `TreeMap` 일 필욘 없어보여요.

Reply:
`Map`으로 하게 되면 더 유연성이 높아질 수 있겠네요!
말씀하신 부분 적용하여 수정했습니다!